lib/AST/ASTPrinter.cpp

```diff
@@ -430,7 +430,7 @@ hasMergeGroup(MergeGroupKind Kind) {
   }
   return false;
 }
-} // end anonymous namespace
+} // namespace swift
```

Comment:
@jrose-apple Excluded the changes to that file from this PR to make this PR mergeable.